src/components/form/form.component.js

```diff
@@ -135,11 +138,10 @@ Form.propTypes = {
   errorCount: PropTypes.number,
 
   /** The total number of warnings present in the form */
-  warningCount: PropTypes.number
-};
+  warningCount: PropTypes.number,
 
-Form.defaultProps = {
-  buttonAlignment: 'right'
+  /** Multipler for vertical spacing between fields, times theme.spacing */
+  fieldSpacingMultiplier: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 7])
```

Comment:
How about this description and prop name? : 

/** Spacing between form fields, given number will be multiplied by base spacing unit (8) */
 fieldSpacing?: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 7]